diff -up jpeg-xl-v0.7.x/tools/libjxl_test.c.omv~ jpeg-xl-v0.7.x/tools/libjxl_test.c
--- jpeg-xl-v0.7.x/tools/libjxl_test.c.omv~	2022-09-09 00:45:59.304206925 +0200
+++ jpeg-xl-v0.7.x/tools/libjxl_test.c	2022-09-09 00:47:08.252017302 +0200
@@ -9,7 +9,8 @@
 
 #include "jxl/decode.h"
 
-int main() {
+int main(int argc, char** argv) {
+  (void) argc; (void) argv;
   if (!JxlDecoderVersion()) return 1;
   JxlDecoder* dec = JxlDecoderCreate(NULL);
   if (!dec) return 1;
